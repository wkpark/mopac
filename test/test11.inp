  pm3 oldens symmetry gnorm=0  pl  1scf
  Formaldehyde  Check the PLS to ensure that the density matrix was read
in correctly
  O    0.0000000  0      0.000000  0      
  C    1.2020463  1      0.000000  0      
  H    1.0912524  1    121.774563  1      
  H    1.0912524  0    121.774563  0    180.000000  0    2    1    3      

   3  1    4
   3  2    4

