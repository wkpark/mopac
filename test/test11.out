 *******************************************************************************
 ** FRANK J. SEILER RES. LAB., U.S. AIR FORCE ACADEMY, COLO. SPGS., CO. 80840 **
 *******************************************************************************

                                 PM3 CALCULATION RESULTS


 *******************************************************************************
 *          MOPAC:  VERSION  6.00               CALC'D. 31-OCT-90               
 *  SYMMETRY - SYMMETRY CONDITIONS TO BE IMPOSED
 *   T=      - A TIME OF  3600.0 SECONDS REQUESTED
 *  DUMP=N   - RESTART FILE WRITTEN EVERY  3600.0 SECONDS
 *  1SCF     - DO 1 SCF AND THEN STOP 
 *  PM3      - THE PM3 HAMILTONIAN TO BE USED
 *  OLDENS   - INITIAL DENSITY MATRIX READ OF DISK
 *  GNORM=   - EXIT WHEN GRADIENT NORM DROPS BELOW .000E+00
 *   PL      - MONITOR CONVERGANCE IN DENSITY MATRIX
 ***********************************************************************040BY040



     PARAMETER DEPENDENCE DATA

        REFERENCE ATOM      FUNCTION NO.    DEPENDENT ATOM(S)
            3                  1             4
            3                  2             4

             DESCRIPTIONS OF THE FUNCTIONS USED

   1      BOND LENGTH    IS SET EQUAL TO THE REFERENCE BOND LENGTH   
   2      BOND ANGLE     IS SET EQUAL TO THE REFERENCE BOND ANGLE    
  PM3 OLDENS SYMMETRY GNORM=0  PL  1SCF
  Formaldehyde  Check the PLS to ensure that the density matrix was read
 in correctly

    ATOM   CHEMICAL  BOND LENGTH    BOND ANGLE     TWIST ANGLE
   NUMBER  SYMBOL    (ANGSTROMS)     (DEGREES)      (DEGREES)
    (I)                  NA:I          NB:NA:I      NC:NB:NA:I     NA   NB   NC

      1      O       
      2      C         1.20205  *                                  1
      3      H         1.09125  *     121.77456  *                 2    1
      4      H         1.09125        121.77456     180.00000      2    1    3


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         O        0.0000    0.0000    0.0000
     2         C        1.2020    0.0000    0.0000
     3         H        1.7767    0.9277    0.0000
     4         H        1.7767   -0.9277    0.0000
  H: (PM3): J. J. P. STEWART, J. COMP. CHEM. 10, 209 (1989).                    
  C: (PM3): J. J. P. STEWART, J. COMP. CHEM. 10, 209 (1989).                    
  O: (PM3): J. J. P. STEWART, J. COMP. CHEM. 10, 209 (1989).                    


          RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =  6


            INTERATOMIC DISTANCES
0
                  O  1       C  2       H  3       H  4
 ------------------------------------------------------
     O    1   0.000000
     C    2   1.202046   0.000000
     H    3   2.004298   1.091252   0.000000
     H    4   2.004298   1.091252   1.855405   0.000000
 ITERATION  1 PLS= 0.100E+01 0.000E+00 ENERGY     -34.0816058 DELTAE -134.0816058
 ITERATION  2 PLS= 0.959E+00 0.000E+00 ENERGY     -34.0816058 DELTAE    0.0000000
 ITERATION  3 PLS= 0.168E-06 0.000E+00 ENERGY     -34.0816058 DELTAE    0.0000000
 ITERATION  4 PLS= 0.364E-07 0.000E+00 ENERGY     -34.0816058 DELTAE    0.0000000

 -------------------------------------------------------------------------------
  PM3 OLDENS SYMMETRY GNORM=0  PL  1SCF
  Formaldehyde  Check the PLS to ensure that the density matrix was read
 in correctly


     1SCF WAS SPECIFIED, SO BFGS WAS NOT USED                 
     SCF FIELD WAS ACHIEVED                                   


                                PM3    CALCULATION
                                                       VERSION  6.00
                                                       31-OCT-90               




          FINAL HEAT OF FORMATION =        -34.08161 KCAL


          TOTAL ENERGY            =       -442.70830 EV
          ELECTRONIC ENERGY       =       -834.62485 EV
          CORE-CORE REPULSION     =        391.91656 EV

          IONIZATION POTENTIAL    =         10.62974
          NO. OF FILLED LEVELS    =          6
          MOLECULAR WEIGHT        =     30.026


          SCF CALCULATIONS  =                1
          COMPUTATION TIME =   0.640 SECONDS





    ATOM   CHEMICAL  BOND LENGTH    BOND ANGLE     TWIST ANGLE
   NUMBER  SYMBOL    (ANGSTROMS)     (DEGREES)      (DEGREES)
    (I)                  NA:I          NB:NA:I      NC:NB:NA:I     NA   NB   NC

      1      O       
      2      C         1.20205  *                                  1
      3      H         1.09125  *     121.77456  *                 2    1
      4      H         1.09125        121.77456     180.00000      2    1    3


            INTERATOMIC DISTANCES
0
                  O  1       C  2       H  3       H  4
 ------------------------------------------------------
     O    1   0.000000
     C    2   1.202046   0.000000
     H    3   2.004298   1.091252   0.000000
     H    4   2.004298   1.091252   1.855405   0.000000


                  EIGENVALUES

 -38.12808 -24.22322 -17.14200 -16.39925 -14.33004 -10.62974   0.82745   2.70755
   4.16117   6.66961


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

         ATOM NO.   TYPE          CHARGE        ATOM  ELECTRON DENSITY
           1          O          -0.3101          6.3101
           2          C           0.2971          3.7029
           3          H           0.0065          0.9935
           4          H           0.0065          0.9935
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     1.826     0.000     0.000     1.826
 HYBRID         0.337     0.000     0.000     0.337
 SUM            2.163     0.000     0.000     2.163


          CARTESIAN COORDINATES 

    NO.       ATOM               X         Y         Z

     1         O                  0.0000    0.0000    0.0000
     2         C                  1.2020    0.0000    0.0000
     3         H                  1.7767    0.9277    0.0000
     4         H                  1.7767   -0.9277    0.0000


          ATOMIC ORBITAL ELECTRON POPULATIONS

   1.85751   1.24388   1.91839   1.29033   1.18061   0.88707   0.92551   0.70967
   0.99352   0.99352
  CHANNEL 12 OPENED



 TOTAL CPU TIME:             0.93 SECONDS

 == MOPAC DONE ==
